sr_robot_launch/scripts/construct_robot_description

```diff
@@ -54,6 +61,18 @@ class SrConstructRobotDescription():
         rospy.set_param('robot_description', urdf)
         rospy.set_param('robot_description_ready', "true")
 
+    def _wait_for_arm_ip_params(self, timeout_in_secs):
+        start_time = rospy.get_time()
+        while not rospy.has_param("/la_sr_ur_robot_hw/robot_ip") and not rospy.has_param("/ra_sr_ur_robot_hw/robot_ip"):
+            if (rospy.get_time() - start_time > timeout_in_secs):
```

Comment:
I wonder if it makes sense to sleep a little bit while waiting. Busy waiting will consume 100% CPU and further delay startup.

Reply:
addressed in https://github.com/shadow-robot/sr_interface/pull/586/commits/df44fa98bba485f979ea12e3ef69c766f53209fa